libmdnsd/CMakeLists.txt

```diff
@@ -95,7 +95,3 @@ install(
     FILES ${PROJECT_BINARY_DIR}/mdnsd_config.h
     DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}"
```

Comment:
${CMAKE_INSTALL_INCLUDEDIR} generates share directory for install targets. Usually it is a good idea to use it, however conan-central-index does not allow for it.  I need to look up the variable that points to root install dir